src/gatherers/accessibility.js

```diff
@@ -21,7 +21,7 @@
 const Gather = require('./gather');
 const fs = require('fs');
 const path = require('path');
-const axe = fs.readFileSync(path.join(__dirname, '../node_modules/axe-core/axe.min.js'));
+const axe = fs.readFileSync(path.join(__dirname, '../../node_modules/axe-core/axe.min.js'));
```

Comment:
Yup. I'm happy to add that as a followup today. I just wanted to do bare-minimum changes in this particular PR.